lib/src/base-styles/_nx-forms.scss

```diff
@@ -10,88 +10,83 @@
 
 .nx-form {
   @include container-vertical;
-  border-top: $nx-border-dark;
-  margin: 0;
-  padding-top: 20px;
+  margin: 0 0 $nx-spacing-xl 0;
   position: relative; // to make a containing block for validation popovers
```

Comment:
We don't have validation popovers anymore.

Reply:
> We don't have validation popovers anymore.

https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/9b23a8ef7f13cbbd2845202a566be6930a420445